Assets/StreamingAssets/Quests/N0B11Y18.txt

```diff
@@ -53,10 +53,10 @@ Message:  1020
 Message:  1021
 %qdt:
  I agreed to spare _rebel_'s life.
- %g said I should just tell _questgiver_
+ He said I should just tell _questgiver_
```

Comment:
Can't Baltham appear as female?

Reply:
I've never known there to be a female Baltham Greyman. The enemy sprite can be male or female, but the flat is always of a male (it's a pre-defined NPC, like Gothryd or Barenziah). I had to fix those pronouns because DFU was trying to assign a female gender to the male Baltham Greyman. One alternative is to fix him with the "male" attribute, but I remember doing that a long time ago and it caused weirdness to happen elsewhere.